File: js/7 kyu/Vowel Count.js
//https://www.codewars.com/kata/54ff3102c1bad923760001f3/train/javascript

function getCount(str) {
  let vowels = ["a", "e", "i", "o", "u"];
  return [...str].filter((element) => vowels.includes(element)).length;
}

function getCount(str) {
  return (str.match(/[aeiou]/ig)||[]).length;
}

getCount("abracadabra"); //5

File: Codewars/cs/7 kyu/Vowel Count.cs
//https://www.codewars.com/kata/54ff3102c1bad923760001f3/train/csharp

using System;
using System.Linq;

public static class Kata
{
    public static int GetVowelCount(string str)
    {
        return str.Count(x => "aeiou".Contains(x));
    }
}
